### ISSUE TITLE
Update our UI/UX to incorporate the Get-Started flow

### ISSUE THREAD

**Issue by y04nqt** [harmony-one/bounties#80]
# Description
Let's improve our UX! Currently, our UX is very scrappy and hard to follow. As I have been demoing harmony to people I've found some good spots to improve or create new areas of interest. We have a lot of room to update links and pages simply and effectively to create a better experience for newcomers.

## Context
As I've been doing demos, oftentimes, I forget where to go for every part of my presentation off the bat. I think it would be extremely helpful to reorganize our links a bit to have a get-started flow that's easy to follow. I've been debating different entry points and have a few good ones in mind to knock out. We have a few different approaches that can synergize together well here but would first want to discuss and decide execution plan before laying it out here.

# Acceptance Criteria
* [ ] Update our UI in docs and landing page to incorporate Get Started UX
* [ ] We should be able to create a system as follows A->B->C->Done->Checkout These Other Opportunities in the Network
* [ ] shouldn't disrupt or confuse the existing UX, this must improve what we currently have

# Reward
[amount of tokens in ONE]

**Comment by Abadiq313** [comment]
> # Description
> Let's improve our UX! Currently, our UX is very scrappy and hard to follow. As I have been demoing harmony to people I've found some good spots to improve or create new areas of interest. We have a lot of room to update links and pages simply and effectively to create a better experience for newcomers.
> 
> ## Context
> As I've been doing demos, oftentimes, I forget where to go for every part of my presentation off the bat. I think it would be extremely helpful to reorganize our links a bit to have a get-started flow that's easy to follow. I've been debating different entry points and have a few good ones in mind to knock out. We have a few different approaches that can synergize together well here but would first want to discuss and decide execution plan before laying it out here.
> 
> # Acceptance Criteria
> * [ ]  Update our UI in docs and landing page to incorporate Get Started UX
> * [ ]  We should be able to create a system as follows A->B->C->Done->Checkout These Other Opportunities in the Network
> * [ ]  shouldn't disrupt or confuse the existing UX, this must improve what we currently have
> 
> # Reward
> [amount of tokens in ONE]

**Comment by armandoaffonso** [comment]
Hello! Is this bounty still open? How can I see what is this about (links screens to be designed) and how to propose a reward?
